modules/acm/src/main/java/edu/gemini/epics/acm/CaObserveSenderImpl.java

```diff
@@ -129,27 +129,43 @@ public CaObserveSenderImpl(String name, String applyRecord, String carRecord, St
             }
         });
 
+        ReadOnlyClientEpicsChannel<Short> stopMark = null;
         if(stopCmd!=null && stopCmd.length()>0) {
-            stopMark = epicsReader.getShortChannel(stopCmd + CAD_MARK_SUFFIX);
-            stopMark.registerListener(stopMarkListener = (String arg0, List<Short> newVals) -> {
-                if (newVals != null && !newVals.isEmpty()) {
-                    CaObserveSenderImpl.this.onStopMarkChange(newVals.get(0));
-                }
-            });
+            try {
+                stopMark = epicsReader.getShortChannel(stopCmd + CAD_MARK_SUFFIX);
+            } catch(Throwable e) {
+                LOG.warn(e.getMessage());
+            }
+            if(stopMark!=null) {
+                stopMark.registerListener(stopMarkListener = (String arg0, List<Short> newVals) -> {
+                    if (newVals != null && !newVals.isEmpty()) {
+                        CaObserveSenderImpl.this.onStopMarkChange(newVals.get(0));
+                    }
+                });
+            }
```

Comment:
I know we still need it, but it feels so strange to see Java code in the ocs3 code base.
I would suggest making `stopMark` an `Optional<ReadOnlyClientEpicsChannel<Short>>` so you can get rid of the `null` check.